### PR TITLE
CMR-10023: Part 1

### PR DESCRIPTION
**Pull request by jmaeng72**
# Overview

### What is the feature/fix?

Change logs from info to debug for search api and data folders

### What is the Solution?

Switch logs from info to debug for logs that are already repeated in the REPORT log

### What areas of the application does this impact?

Search App logs

# Checklist

- [ ] I have updated/added unit and integration tests that prove my fix is effective or that my feature works
- [ ] New and existing unit and int tests pass locally and remotely with my changes
- [x] I have removed unnecessary/dead code and imports in files I have changed
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have cleaned up integration tests by doing one or more of the following:
  - de-duped, consolidated, removed dead int tests
  - transformed applicable int tests into unit tests
  - refactored to reduce number of system state resets by updating fixtures (use-fixtures :each (ingest/reset-fixture {})) to be :once instead of :each

### CODE REVIEW

File: search-app/src/cmr/search/api/concepts_lookup.clj [hunk 1]
```diff
@@ -73,13 +73,13 @@
 (defn- find-concept-by-concept-id*
   "Perfrom the retrieval of concept by concept id and revision id"
   ([ctx result-format concept-id]
-   (info (format "Search for concept with cmr-concept-id [%s]" concept-id))
+   (debug (format "Search for concept with cmr-concept-id [%s]" concept-id))
```
Comment (isja17) [inline search-app/src/cmr/search/api/concepts_lookup.clj:76]: Putting this comment here but really it applies to most of these changes.

Removing the start log is an interesting approach. I _think_ that we don't use them in dashboards, reports, etc - a quick look through some dashboards showed that we use the "found x <concept> in y ms" logs. We need to double, triple check that that's true before merging this.  I am a little bit concerned about losing end-to-end visibility of a searches lifecycle, but that might be fine.

File: search-app/src/cmr/search/api/tags.clj [hunk 2]
```diff
@@ -101,7 +99,7 @@
   (verify-tag-modification-permission context :update)
   (common-enabled/validate-write-enabled context "search")
   (validate-tag-content-type headers)
-  (info (format "Tagging [%s] on collections by query: %s by client: %s."
+  (debug (format "Tagging [%s] on collections by query: %s by client: %s."
```
[the remark this answers is not in the document]
Comment (isja17) [reply search-app/src/cmr/search/api/tags.clj:102]: that's a reason to cut it imo. we might be surprised by how much we gain by cutting out all of these small logs that only show up a handful of times a day. i would bet we have a lot of them throughout all of apps, and they are mostly noise.

File: search-app/src/cmr/search/api/concepts_lookup.clj (hunk 1 above)
Comment (zimzoom) [reply search-app/src/cmr/search/api/concepts_lookup.clj:76]: IIRC the "found x in y ms" logs only get created when results are found. So searches that don't find anything or error out won't get captured

File: search-app/src/cmr/search/api/tags.clj (hunk 2 above)
[the remark this answers is not in the document]
Comment (zimzoom) [reply search-app/src/cmr/search/api/tags.clj:102]: seconded; logs should be cut across the board based on usefulness alone and not predicted impact to target. Minimalism for the long game and quality of life.

File: search-app/src/cmr/search/api/concepts_lookup.clj (hunk 1 above)
Comment (zimzoom) [reply search-app/src/cmr/search/api/concepts_lookup.clj:76]: But I think the access logs should cover it

**Review by zimzoom — APPROVED** [review]

**Review by jceaser — APPROVED** [review]

File: search-app/src/cmr/search/api/concepts_lookup.clj (hunk 1 above)
Comment (jmaeng72) [reply search-app/src/cmr/search/api/concepts_lookup.clj:76]: I went through the Splunk dashbaords and alerts and did not find any query that uses 'Search for concept...' I only found the one using 'Searching for...'

Ad for the actual info, this is already captured in the REPORT log

Comment (isja17) [reply search-app/src/cmr/search/api/concepts_lookup.clj:76]: > IIRC the "found x in y ms" logs only get created when results are found. So searches that don't find anything or error out won't get captured

Yeah, that's what my comment was saying - we don't use these for reports. IIRC, john had some reason to not get rid of this particular log though, right?

Comment (jmaeng72) [reply search-app/src/cmr/search/api/concepts_lookup.clj]: I will put these logs back

**Pull request merged**